src/zcl_abapgit_factory.clas.abap

```diff
@@ -57,6 +57,11 @@ CLASS zcl_abapgit_factory DEFINITION
     CLASS-METHODS get_cts_api
       RETURNING
         VALUE(ri_cts_api) TYPE REF TO zif_abapgit_cts_api.
+    CLASS-METHODS get_fe_services
```

Comment:
"get_frontend_services" is only 22 characters, I think the max is 30?

Reply:
renamed